ocs_ci/ocs/resources/pod.py

```diff
@@ -308,6 +308,42 @@ def get_ceph_tools_pod():
     return ceph_pod
 
 
+def get_rbd_provisioner_pod():
+    """
+    Get the RBD provisioner pod
+
+    Returns:
+        Pod object: The RBD provisioner pod object
+    """
+    ocp_pod_obj = OCP(
```

Comment:
we just need to create a PVC right? why create POD?

Reply:
The logic for sampling the PVC creation is through the provisioner logs